src/data_source/storage/sql.rs

```diff
@@ -174,11 +175,8 @@ fn add_custom_migrations(
 
 /// Postgres client config.
 #[derive(Clone, Debug)]
-pub struct Config<DB>
-where
-    DB: Database,
-{
-    db_opt: <DB::Connection as Connection>::Options,
+pub struct Config {
+    db_opt: PgConnectOptions,
```

Comment:
So we would need two Configs when we make `Sqlite` change with that feature thing?

Reply:
Okay, I will revert this change then in Sqlite PR